watson_developer_cloud/tone_analyzer_v3.py

```diff
@@ -19,24 +19,25 @@
 
 
 from watson_developer_cloud.watson_developer_cloud_service import WatsonDeveloperCloudService
-
+import copy
 
 class ToneAnalyzerV3(WatsonDeveloperCloudService):
     default_url = 'https://gateway.watsonplatform.net/tone-analyzer/api'
-    latest_version = '2016-02-11'
+    latest_version = '2016-05-19'
 
-    def __init__(self, version, url=default_url, username=None, password=None, use_vcap_services=True):
+    def __init__(self, version=latest_version, url=default_url, username=None, password=None, use_vcap_services=True):
```

Comment:
@herchu we don't use a default `version_date`. It's a required parameter and users should always provide a valid value